src/custodian/src/custodian_server.erl

```diff
@@ -143,52 +143,7 @@ handle_db_event(_DbName, _Event, _St) ->
     {ok, nil}.
 
 check_shards() ->
-    [send_sensu_event(Item) || Item <- custodian:summary()].
```

Comment:
this is a cut too far imo. we still want to log the descriptive error message to couch_log. the pluggable module allows a sysadmin to send that message (or one of its own invention) to some other system.

Reply:
Hah, I didn't even notice that logging cleverly interleaved in the case statement. In any case, I've returned them, and changed the `send_event/3` callback to DRY things up a bit.